cluster/src/main/java/com/hartwig/pipeline/resource/Hg37ResourceFiles.java

```diff
@@ -0,0 +1,52 @@
+package com.hartwig.pipeline.resource;
+
+import static com.hartwig.pipeline.resource.ResourceNames.AMBER_PON;
+import static com.hartwig.pipeline.resource.ResourceNames.BEDS;
+import static com.hartwig.pipeline.resource.ResourceNames.GC_PROFILE;
+import static com.hartwig.pipeline.resource.ResourceNames.GRIDSS_REPEAT_MASKER_DB;
+import static com.hartwig.pipeline.resource.ResourceNames.MAPPABILITY;
+import static com.hartwig.pipeline.resource.ResourceNames.REFERENCE_GENOME;
+import static com.hartwig.pipeline.resource.ResourceNames.SAGE;
+import static com.hartwig.pipeline.resource.ResourceNames.SNPEFF;
+
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+
+public class Hg37ResourceFiles implements ResourceFiles
+{
+    public static final String HG37_DIRECTORY = "hg37";
+
+    public RefGenomeVersion version() { return RefGenomeVersion.HG37; }
+
+    public String versionDirectory() { return HG37_DIRECTORY; }
+
+    private String formPath(String name, String file) {
+        return String.format("%s/%s/%s/%s", VmDirectories.RESOURCES, name, versionDirectory(), file);
+    }
+
+    private static final String REF_GENOME_FASTA_HG37_FILE = "Homo_sapiens.GRCh37.GATK.illumina.fasta";
+
+    public static final String REFERENCE_GENOME_FASTA_HG37 =
+            String.format("%s/%s/%s/%s", VmDirectories.RESOURCES, REFERENCE_GENOME, HG37_DIRECTORY, REF_GENOME_FASTA_HG37_FILE);
```

Comment:
Not sure why there is public static method. I think it makes more sense for the batch applications to access this via the ResourcesFilesFactory same as the normal pipeline.

Reply:
Removed this method